app/src/main/java/net/squanchy/about/licenses/LicensesActivity.java

```diff
@@ -0,0 +1,55 @@
+package net.squanchy.about.licenses;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.view.MenuItem;
+
+import net.squanchy.R;
+import net.squanchy.fonts.TypefaceStyleableActivity;
+import net.squanchy.support.view.CardSpacingItemDecorator;
+
+public class LicensesActivity extends TypefaceStyleableActivity {
+
+    private RecyclerView licensesList;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_licenses);
+        setupToolbar();
+
+        licensesList = (RecyclerView) findViewById(R.id.libraries_list);
+    }
+
+    private void setupToolbar() {
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setDisplayShowTitleEnabled(true);
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+
+        licensesList.setAdapter(new LibrariesAdapter(this));
```

Comment:
just asking: why are we creating the `LibrariesAdapter` every time the `Activity` is started? This is a fixed list which is never going to change at runtime so wouldn't it be better to set it in the `onCreate` ? Same for the `LayoutManager` and the `ItemDecorator`